python/lsst/atmospec/spectraction.py

```diff
@@ -213,11 +214,18 @@ def _translateCentroid(dmXpos, dmYpos):
         if parameters.DEBUG:
             self.debugPrintTargetCentroidValue(image)
 
-        self._setReadNoiseFromExp(image, exp, 1)
+        self._setReadNoiseFromExp(image, exp, 8.5)
         # xxx remove hard coding of 1 below!
-        image.gain = self._setGainFromExp(image, exp, .85)  # gain required for calculating stat err
+        import lsst.daf.butler as dafButler
+        # butler = dafButler.Butler("/sdf/group/rubin/repo/oga/", collections=['LATISS/calib','LATISS/raw/all'])
+        butler = dafButler.Butler("/repo/embargo", collections=['LATISS/calib','LATISS/raw/all'])
```

Comment:
This initialisation of the butler with hardcoded paths is probably not what I should do but I don't know how to make it nicer.